server/sonar-web/src/main/js/apps/overview/meta/MetaQualityGate.tsx

```diff
@@ -17,26 +17,29 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import React from 'react';
+import * as React from 'react';
 import { Link } from 'react-router';
 import { translate } from '../../../helpers/l10n';
 import { getQualityGateUrl } from '../../../helpers/urls';
 
-const MetaQualityGate = ({ gate, organization }) => {
+interface Props {
+  organization?: string;
+  qualityGate: { isDefault?: boolean; key: string; name: string };
+}
+
+export default function MetaQualityGate({ qualityGate, organization }: Props) {
   return (
     <div className="overview-meta-card">
       <h4 className="overview-meta-header">{translate('overview.quality_gate')}</h4>
 
       <ul className="overview-meta-list">
         <li>
-          {gate.isDefault && (
+          {qualityGate.isDefault && (
             <span className="note spacer-right">{'(' + translate('default') + ')'}</span>
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use a template literal instead of concatenating with a string literal. ([typescript:S3512](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3512))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Foverview-sidebar&issues=AWFwELMXh1v1MSKKr4AH&open=AWFwELMXh1v1MSKKr4AH)

[ISSUE_KEY]: AWFwELMXh1v1MSKKr4AH